ui/values/localizable/fr.go

```diff
@@ -24,7 +24,7 @@ const FR = `
 "resumeAccountDiscoveryTitle" = "Déverouiller pour reprendre la restauration";
 "unlockWithPassword" = "Déverrouiller avec mot de passe"
 "offline" = "Hors ligne, ";
-"showDetails" = "Montrer les détails";
+"viewDetails" = "View details";
```

Comment:
This should be translated to French

Reply:
it doesn't necessarily have to, we would have a translator do all non-english translations

Reply:
The idea is to have a place holder for the other languages pending when we do a general sting localization like the https://github.com/planetdecred/godcr/pull/906